README.md

```diff
@@ -537,3 +537,75 @@ Prio 2:
 - review the database indices and the foreign keys
 - include a list of basic values in test.php e.g. CO2 of rice
 - allow personal groups up to 100 persons and to join up 20 named groups
+
+---
+
+
+## Setup Guide 
```

Comment:
I guess this should be linked to the existing docker installation as "nginx docker"